internal/images/images.go

```diff
@@ -36,14 +45,35 @@ func getDockerConfigPath() string {
 	return filepath.Join(home, ".docker", "config.json")
 }
 
+func GetFriendlyErrorMessage(err error, pods []string) string {
+	if err == nil {
+		return ""
+	}
+
+	errorMessage := err.Error()
+	switch {
+	case containsAnyOf(errorMessage, []string{"authentication", "auth", "authorized"}):
+		return "|| Authentication error."
+	case containsAnyOf(errorMessage, []string{"no image found", "image not found"}):
+		return "|| Image not found. Some pods like `" + pods[0] + "` are using an image that no longer exists."
+	case containsAnyOf(errorMessage, []string{"no such host"}):
+		return "|| communication error, check your url host."
+	default:
+		return "|| An unknown error occurred. Please run in debug mode using the flag '-d' for more details."
```

Comment:
The use of the `||` in the text seems strange to me, looking like a pipe. Is there a particular reason for this format? If not then it would be good to change.